.craft.yml

```diff
@@ -0,0 +1,29 @@
+minVersion: "0.9.6"
+github:
+  owner: getsentry
+  repo: sentry-native
+changelogPolicy: simple
+
+statusProvider:
+  name: github
+artifactProvider:
+  name: zeus
+
+targets:
+  - name: github
```

Comment:
so this pushes the generated zip file also to the github release, right? How does it do the naming?

Reply:
This publishes to GitHub and applies the Changelog, correct. Naming of the file is always `sentry-native.zip`. Do you think it's worth keeping the version? Would have to figure out how to get the version in Azure.